Firebase/Storage/FIRStorageTask.m

```diff
@@ -26,22 +26,16 @@
 
 @implementation FIRStorageTask
 
```

Comment:
Fix Storage/FIRStorageTask.m:27:17: warning: method override for the designated initializer of the superclass '-init' not found [-Wobjc-designated-initializers]

Reply:
Done